docs/tutorial.md

```diff
@@ -426,7 +425,7 @@ To remove specific listener you can use `bot.removeListener()` method.
 - `bot.once(eventName, listener)`
   Execute the `listener` function, only once, the first time the event named `eventName` triggered.
 - `bot.removeListener(eventName, listener)`
-  Removes the specified `listener` for the event named `eventName`.
+  Removes the specified `listener` for the event named `eventName`. This will not work with annonymous functions, as they don't have a name.
```

Comment:
It's a bit confusing, `const a = function () {}` is still an anonymous function.

Reply:
Ow wait it is? So passing it `a` won't remove the listener?

Reply:
`const a = function() {}` and `const a = () => {}` are both function expressions (the variable a gets assigned an anonymous function. In both case it will not remove the listener.

`function a() {}` is a function declaration, this is what you need to be able to remove the listener.

https://www.w3schools.com/js/js_function_definition.asp

Reply:
Well, TIL

I'll mention it in the tutorial.

Reply:
> Ow wait it is? So passing it `a` won't remove the listener?

passing `a` will remove `a` listner. I meant that it's been described in a bit confusing way. What you said there was that even if we assign an anonymous function to a variable and pass that variable it won't work. But it will.

https://nodejs.org/api/events.html#events_emitter_removelistener_eventname_listener
Even example in node api shows so. 

Idk what you meant @Karang, that it won't work.